### PR TITLE
DOCSP-27021 add doc for tree explorer document context menu

### PR DESCRIPTION
**Pull request by kanchana-mongodb**
## DESCRIPTION
Adds documentation for new actions in the context menu for documents in the explorer view.

## STAGING
https://preview-mongodbkanchanamongodb.gatsbyjs.io/mongodb-vscode/DOCSP-27021/databases-collections/#view-collection-documents-and-schema

## JIRA
https://jira.mongodb.org/browse/DOCSP-27021

## BUILD LOG
https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=65d4e1bc71111659827980ad (clean)

## Self-Review Checklist

- [ ] Is this free of any warnings or errors in the RST?
- [ ] Is this free of spelling errors?
- [ ] Is this free of grammatical errors?
- [ ] Is this free of staging / rendering issues?
- [ ] Are all the links working?

## External Review Requirements

[What's expected of an external reviewer?](https://wiki.corp.mongodb.com/display/DE/Reviewing+Guidelines+for+the+MongoDB+Server+Documentation)

### CODE REVIEW

**Review by Anemy — APPROVED** [review]
lgtm!

File: source/databases-collections.txt [hunk 1]
```diff
@@ -63,11 +68,41 @@ You may edit this single document:
...
+#. Click the :guilabel:`Play Button` in VS Code's top navigation bar to
+   insert this document in the collection.
+#. Click :guilabel:`Yes` to confirm. 
+
+   The :guilabel:`Playground Result` window displays the inserted ID of
+   the new document. 
+
+*To remove a document from the collection*, you can: 
+
+1. Right-click the ID of the document that document you want to remove.
```
Comment (jocelyn-mendez1) [inline source/databases-collections.txt:102]: [typo]

```suggestion
1. Right-click the ID of the document you want to remove.
```

File: source/databases-collections.txt [hunk 2]
```diff
@@ -63,11 +68,41 @@ You may edit this single document:
...
 
-To view all of the collection's documents in an array, you can:
+*To view all of the collection's documents* in an array, you can:
 
 1. Right-click a collection.
 2. Click :guilabel:`View Documents`.
 
+*To copy a document in the collection*, you can: 
+
+1. Right-click the ID of the document that document you want to copy.
```
Comment (jocelyn-mendez1) [inline source/databases-collections.txt:78]: [typo]

```suggestion
1. Right-click the ID of the document you want to copy.
```

File: source/databases-collections.txt [hunk 3]
```diff
@@ -63,11 +68,41 @@ You may edit this single document:
...
+2. Click :guilabel:`Copy Document`.
+
+   |vsce| copies the document to your clipboard.
+
+*To clone a document in the collection*, you can: 
+
+1. Right-click the ID of the document that document you want to clone.
```
Comment (jocelyn-mendez1) [inline source/databases-collections.txt:85]:
```suggestion
1. Right-click the ID of the document you want to clone.
```

**Review by jocelyn-mendez1 — APPROVED** [review]
Thanks for the updates @kanchana-mongodb, this LGTM!

**Pull request merged**